tests/helpers/mod.rs

```diff
@@ -118,3 +122,75 @@ pub fn create_local_table(schema: HashMap<&str, &str>, partitions: Vec<&str>) ->
 
     path
 }
+
+pub fn create_kdi(
+    app_id: &str,
+    topic: &str,
+    table: &str,
+    allowed_latency: u64,
+    max_messages_per_batch: usize,
+    min_bytes_per_file: usize,
+) -> KafkaJsonToDelta {
+    env::set_var("AWS_S3_LOCKING_PROVIDER", "dynamodb");
+    env::set_var("DYNAMO_LOCK_TABLE_NAME", "locks");
+    env::set_var("DYNAMO_LOCK_OWNER_NAME", Uuid::new_v4().to_string());
+    env::set_var("DYNAMO_LOCK_PARTITION_KEY_VALUE", "emails_s3_tests");
```

Comment:
makes sense to use the same lock table ("emails_s3_tests") defined in setup for all tests, but we might consider changing the name of it.

Reply:
ugh, we definitely should use the same table but not the `DYNAMO_LOCK_PARTITION_KEY_VALUE `, that's a leftover from copypasdting, gonna fix asap